com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/HDGlobalSettingsWindow.Skin.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering.HighDefinition;
+using UnityEngine.Rendering;
+using UnityEngine.UIElements;
+using UnityEditorInternal;
+using System.Linq;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    using CED = CoreEditorDrawer<SerializedHDRenderPipelineGlobalSettings>;
+
+    internal partial class HDGlobalSettingsPanelIMGUI
+    {
+        public class Styles
```

Comment:
minor: Normaly we set the class `internal` and the `static` fields to `public`

Reply:
oh, they I will update them, thanks for noticing.

Reply:
wait so should we have : `public static class Styles` instead?

Reply:
No it should be (internal) static class Style with public field inside. It is not a big deal now because HDGlobalSettingsPanelIMGUI is already internal but if the visibility change, we can not see this.

Only exception is for shared style like Light and Camera that have field common in several SRP.